hbase-testing-util/src/main/java/org/apache/hadoop/hbase/testing/TestingHBaseClusterImpl.java

```diff
@@ -147,7 +154,7 @@ public void startHBaseCluster() throws Exception {
   public void start() throws Exception {
     Preconditions.checkState(!miniClusterRunning, "Cluster has already been started");
     if (externalZkConnectString == null) {
-      util.startMiniZKCluster();
+      util.startMiniZKCluster(option.getNumZkServers());
```

Comment:
While testing `getZooKeeperQuorum`, I noticed that `numZkServers` was never used. The fix is trivial, so I included it here, but if you think it deserves a separate issue, I'll split it.